### ISSUE TITLE
beamer overlay specifications for \spfstep and friends

### ISSUE THREAD

**Issue by kohlhase** [slatex/sTeX#381]
It would be great if `\spfstep` and `\begin{subproof}` would accept beamer overlay specifications, so that I can structure proofs in my lectures better. 

Concretely, `notesslides.sty` should probably patch `\spfstep` so that `\spfstep<2-4>` just passes the `<2-4>` to the inner item. 

Alternatively `\spfstep<2-4>{foo}` could expand to `\only<2-4>{\spfstep{foo}` I guess.

**Comment by kohlhase** [comment]
BTW, I could not the `\only` method to work in my example.

**Comment by kohlhase** [comment]
Also I remembered that I could maybe just use `\pause` from beamer, but that also gives me lots of `missing item` errors.

**Comment by Jazzpirate** [comment]
yes, both `\pause` and `\only` would need to non-trivially escape from the spf-environment. I'm not surprised that causes trouble. The spf-environments should probably also redesigned...
